k8s/helm/nestjs-app/Chart.yaml

```
@@ -0,0 +1,32 @@
+  - name: postgresql
+    version: 12.x.x
+    repository: https://charts.bitnami.com/bitnami
+    condition: postgresql.enabled
+  - name: redis
+    version: 18.x.x
+    repository: https://charts.bitnami.com/bitnami
+    condition: redis.enabled
+  - name: kafka
+    version: 26.x.x
+    repository: https://charts.bitnami.com/bitnami
+    condition: kafka.enabled
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)